config.go

```diff
@@ -74,8 +78,8 @@ type ClientBinary struct {
 }
 
 type ClientConfig struct {
-	ClientBinaries []ClientBinary `yaml:"client_binaries"`
-	LatestRevision string         `yaml:"latest_revision"`
+	ClientBinaries []ClientBinary `yaml:"client_binaries" json:"ClientBinaries"`
+	LatestRevision string         `yaml:"latest_revision" json:"LatestRevision"`
```

Comment:
we can't change this to have the same tag as the yaml? I'm not sure what happens if you pass json to the yaml parser, it could get weird.

Reply:
No we cannot. This object was marshalled as json directly by the api server, and since it had no json metadata, it used the Go field names. Changing these will break existing evergreen clients.

Reply:
Should I add a ticket to fix this once and for all in the future? We'll need to wait a month or so while everyone updates their evergreen clients.

Reply:
a ticket sounds fine. Can you have the client attempt to read new and legacy formats? and we'll change the server in a bit? 

Does it make sense to move this structure to apimodels?

Reply:
I've already made APIClientConfig for representing these as JSON objects by the rest server.
It might make sense to move this structure to model/cli_update.go.

As a part of that ticket, we can remove the route in api server that serves this structure, and remove the json tags, since they won't be needed anymore.

How's all that sound?